SU-SOE/stanford_soe_helper
# 7.x 2.x hotfix chameleon

# READY FOR REVIEW


# Summary
- Adding Injector 66 into code. (https://stanfordits.atlassian.net/browse/SOE-3284)
- Fixing the related spotlights view to exclude the current nid
- Removing the "See all" button from spotlight pages (https://stanfordits.atlassian.net/browse/SOE-3283)
- Fixed Postcard span12 class, inappropriately targeted (https://stanfordits.atlassian.net/browse/SOE-3288)

# Needed By (Date)
- Sprint end

# Criticality
- Client is eagerly awaiting this for the release.


# Steps to Test
- Switch to this branch: `7.x-2.x-hotfix-chameleon`
- Revert feature: `drush fr stanford_people_spotlight_layouts`
- Revert feature: `drush fr stanford_people_spotlight_views`
- Clear cache so Drupal can find the module: `drush cc all`
- Go to a spotlight person at: `spotlight/ryan-cohen`
- Make sure the current node is not showing in the related spotlight block at the bottom of the page.
- Make sure the "See all" button does not show SEE: https://stanfordits.atlassian.net/browse/SOE-3283.
- Go to a this page: `faculty-research` and make sure it looks right: https://stanfordits.atlassian.net/browse/SOE-3288

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-3283
https://stanfordits.atlassian.net/browse/SOE-3284
https://stanfordits.atlassian.net/browse/SOE-3288

## Related PRs

## More Information

## Folks to notify
@boznik 


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
